.travis.yml

```diff
@@ -71,6 +88,13 @@ before_script:
         echo "Only docs were updated, stopping build process."
         exit
       fi
+      if [ "${TESTING}" != "docs" ]; then
+        # clang complains about unused-parameter a lot, redirect stderr
```

Comment:
At some point, perhaps the `before_script` should be factored out instead of duplicating all its contents?